nemo/utils/exp_manager.py

```diff
@@ -58,7 +58,8 @@ class CheckpointMisconfigurationError(NeMoBaseException):
 
 @dataclass
 class CallbackParams:
-    filepath: Optional[str] = None  # If None, exp_manager will attempt to handle the filepath
```

Comment:
Instead of removing it, how about keeping it, and making it exclusive to the two new flags? A lot of older training scripts will break otherwise

Reply:
afaik, no one uses this. It stays none the majority of the time and is set by exp_manager.

Reply:
Added it back

Reply:
Thanks !